core/src/avm2/object/context3d_object.rs

```diff
@@ -353,12 +353,27 @@ impl<'gc> Context3DObject<'gc> {
     ) {
         let source = source.read();
 
-        assert_eq!(source.width(), dest.width(), "Mismatched width");
-        assert_eq!(source.height(), dest.height(), "Mismatched height");
+        // Note - Flash appears to allow a source that's larger than the destination.
+        // Let's leave in this assertion to see if there any real SWFS relying on this
+        // behavior.
```

Comment:
This sentence reads a bit weird to me. But I get the meaning, so it's not important.
Anyway, something this might sound a bit better IMHO:
> Let's leave this assertion in to see if there are any real SWFs relying on this behavior.